schema/content.schema.json

```diff
@@ -25,6 +28,15 @@
         "editorOnly": true
       }
     },
+    "_lang": {
+      "type": "string",
+      "title": "Language",
+      "description": "The language/locale e.g. 'en' or 'fr-CA'.",
+      "default": "en",
+      "_backboneForms": {
+        "editorAttrs": { "disabled": true }
```

Comment:
You can also set `_backboneForms.showInUi` to `false` to hide the input completely (not sure which behaviour would be most appropriate).

Reply:
good to know, yep possibly better hidden as lang menu shows currently selected language and user will never be able to edit this field